moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -48,10 +48,14 @@ const validationSchema = yup.object().shape({
   completionDate: yup.date().nullable().optional(),
   description: yup.string(),
   // Signal field is required if the selected component inserts into the feature_signals table
-  signal: yup.object().when("component", {
-    is: (val) => val?.data?.feature_layer?.internal_table === "feature_signals",
-    then: yup.object().required(),
-  }),
+  signal: yup
+    .object()
+    .nullable()
```

Comment:
The form validation Yup resolver is now complaining about `null` for this field when a non-signal component was being edited.